src/com/puppetlabs/puppetdb/config.clj

```diff
@@ -320,25 +330,21 @@
                      (assoc :url-prefix url-prefix)
                      (utils/assoc-when :update-server "http://updates.puppetlabs.com/check-for-updates"))))))
 
-(defn warn-if-sslv3
-  "If the ssl-protocols config is present and contains sslv3, warn the user. Logging
-   at this point may or may not be setup, so warning via *err* and logging is done."
+(defn fix-tk-config
+  "Fix configuration before passing it to trapperkeeper.
+
+   In particular:
+   * Move certificate-whitelist from [jetty] to [global]"
```

Comment:
I think you mean [puppetdb] rather than global here? The code isn't moving it to global